src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int MAX_LENGTH = 5;
+    private static final int MIN_LENGTH = 1;
+
```

Comment:
```suggestion
    private static final int MIN_LENGTH = 1;
    private ColorSupplier colorSupplier = new ColorSupplier();
```
move declaration to a class instead of method